events/start_handler.go

```diff
@@ -32,14 +32,20 @@ func getDNSSearch(container *docker.Container) []string {
 
 	//from labels - for upgraded systems
 	if container.Config.Labels != nil {
-		if value, ok := container.Config.Labels["io.rancher.stack_service.name"]; ok {
-			splitted := strings.Split(value, "/")
-			svc := strings.ToLower(splitted[1])
-			stack := strings.ToLower(splitted[0])
-			svcNameSpace = svc + "." + stack + "." + RancherDomain
-			stackNameSpace = stack + "." + RancherDomain
-			defaultDomains = append(defaultDomains, svcNameSpace)
-			defaultDomains = append(defaultDomains, stackNameSpace)
+		setDNSSearch := true
```

Comment:
nit: can we rename this value to something that reflects the fact that the flag will determine if rancher stack/service domains are gonna be set? like setServiceSearch